ports/esp32/mphalport.c

```diff
@@ -119,7 +119,8 @@ int mp_hal_stdin_rx_chr(void) {
     }
 }
 
-void mp_hal_stdout_tx_strn(const char *str, size_t len) {
+mp_uint_t mp_hal_stdout_tx_strn(const char *str, size_t len) {
+    mp_uint_t ret = len;
```

Comment:
I understand the motivation of trying to keep all of the ports using the same pattern, but in cases where `ret ==  len` always then it's OK to return `len`.

Reply:
I agree.